### ISSUE TITLE
Can't change the Mywords Author after RMcommon update

### ISSUE THREAD

**Issue by JustineBABY** [bitcero/mywords#32]
Hello Edu,

This problem is still here after RMcommon update to version 2.3.6.2, (now 2.3.7.0)

I can't change the author in the mywords post. When I click on the User..., I have alway this message: "Session token expired!" and "OK". And when I click on OK, the page of post edit is automaticaly refreshing.

This is my Xoops config:
Xoops: 2.5.9
RMcommon: 2.3.7.0
PHP: 7.1
MySql: 5.6.33
Mywords: 2.2.6.2

Thanks for reviewing.

**Comment by bitcero** [comment]
Have you updated to MyWords 2.2.6.3?

**Comment by JustineBABY** [comment]
Yes, my Mywords version is now: 2.2.6.3, but it do not function.

**Comment by bitcero** [comment]
Ok. Have you tried with recent update of rmcommon? If yes, please provide me details about the procedure that you follow to change the post author in order to reproduce the error.

**Comment by JustineBABY** [comment]
I think this probleme become the RMcommon, because if I return to the RMcommon version 2.3.4.1, it work as every well.

But now, with the present version of RMcommon, when I click on th [Users...] buton, I have the message in the pop-up windows:

mysite.com say:
Session token expired!

If I click on OK, the page of post (modules/mywords/admin/posts.php) is automaticaly refreshing.

**Comment by JustineBABY** [comment]
the pop up is like this:

![sans-titre-1](https://user-images.githubusercontent.com/16358737/29984039-c5d23ee4-8f58-11e7-9e3b-8374977607dd.jpg)

If I click on OK, the post page is automaticaly refreshing.
